diff --git a/node_modules/@aws-cdk/aws-codebuild/lib/project.js b/node_modules/@aws-cdk/aws-codebuild/lib/project.js
index 059b726..967645d 100644
--- a/node_modules/@aws-cdk/aws-codebuild/lib/project.js
+++ b/node_modules/@aws-cdk/aws-codebuild/lib/project.js
@@ -966,7 +966,7 @@ var BuildEnvironmentVariableType;
     /**
      * An environment variable in plaintext format.
      */
-    BuildEnvironmentVariableType["PLAINTEXT"] = "PLAINTEXT";
+    //BuildEnvironmentVariableType["PLAINTEXT"] = "PLAINTEXT";
     /**
      * An environment variable stored in Systems Manager Parameter Store.
      */
